.github/ISSUE_TEMPLATE/1_bug_report.yaml

```diff
@@ -35,6 +35,7 @@ body:
       label: How to reproduce the bug
       description: >
         Provide steps and example code here.
+        You can also paste a link to a Google Colab, see our [bug report template](https://colab.research.google.com/github/Lightning-AI/lightning/blob/master/examples/pl_bug_report/bug_report_model.ipynb).
```

Comment:
```suggestion
        You can also paste a link to a Google Colab (see our [Colab bug report template](https://colab.research.google.com/github/Lightning-AI/lightning/blob/master/examples/pl_bug_report/bug_report_model.ipynb)) or adapt the following [snippet](https://github.com/Lightning-AI/lightning/blob/master/examples/pl_bug_report/bug_report_model.py).
```

Reply:
@carmocca I propose to keep this short. I will follow up with a bug-report model for apps. We will then need to change this text again.

Reply:
I think this is fine.. with render it will be few more words but atleast user will get the link to the script.